nautobot/core/filters.py

```diff
@@ -629,15 +640,28 @@ def _generate_lookup_expression_filters(cls, filter_name, filter_field):
         if field is None:
             return magic_filters
 
+        try:
+            field_type = cls._meta.model._meta.get_field(field_name)
+        except FieldDoesNotExist:
+            field_type = None
+
+        # If the field allows null values, add an `isnull`` check
+        if field.null and isinstance(field_type, NULLABLE_CLASSES):
+            # Use this method vs extend as the `lookup_map` variable is generally one of
+            # the constants which we do not want to update
+            lookup_map = dict(lookup_map, **{"isnull": "isnull"})
```

Comment:
Would be good to add tests for this logic, likely in `nautobot/core/tests/test_filters.py` `BaseFilterSetTest` class.